Source/santad/SNTExecutionControllerTest.m

```diff
@@ -86,6 +87,24 @@ - (santa_vnode_id_t)getVnodeId {
   return (santa_vnode_id_t){.fsid = 1234, .fileid = 5678};
 }
 
+- (void)checkMetricCounters:(NSString *)expectedFieldValueName expected:(NSNumber *)expectedValue {
+  SNTMetricSet *metricSet = [SNTMetricSet sharedInstance];
+  NSDictionary *eventCounter = [metricSet export][@"metrics"][@"/santa/events"];
+  BOOL foundField;
+  for (NSDictionary *fieldValue in eventCounter[@"fields"][@"action_response"]) {
+    if ([expectedFieldValueName isEqualToString:fieldValue[@"value"]]) {
+      XCTAssertEqualObjects(expectedValue, fieldValue[@"data"], @"%@ counter not incremented",
```

Comment:
nit: maybe counter not equal? For instance the counter could be incremented but not equal to the expected value.

Reply:
Yup good catch that was left over before refactoring this out.